encrypted-media-respec.html

```diff
@@ -3900,6 +3927,21 @@
               </p>
             </dd>
 
+            <dt>Do not expose per-origin information to unrelated entities</dt>
+            <dd>
+              <p>
+                Do not provide <a def-id="origin">origin(s)</a> or values <a def-id="associable"></a> with origins to individualization servers or other entities not related to the origin.
+                Follow the requirements and recommendations in the <a href="#individualization">Individualization</a> section if such a process is used by the implementation. 
+              </p>
+            </dd>
+
+            <dt>Do not expose <a def-id="distinctive-permanent-identifiers"></a> to the application</dt>
+            <dd>
+              <p>
+                Implementations MUST NOT expose <a def-id="distinctive-permanent-identifiers"></a> to the application or origin. 
+              </p>
+            </dd>
+
```

Comment:
The first item is restating the requirements in the Individualization section.

The second item is restating what is said in the DPI definition:
>Distinctive Permanent Identifiers must not ever be exposed to the application, even in encrypted form.